InputTime/index.jsx

```diff
@@ -7,7 +8,7 @@ const leftPadTimeUnit = timeUnit => (timeUnit < 10 ? `0${timeUnit}` : timeUnit);
 /* eslint-disable react/prop-types */
 
 const renderAmPm = ({ value, onChange, submitting }) =>
-  <select
+  (<select
```

Comment:
Feels smooth to me. Perhaps we could update our ESLint to ignore this multiple lines required `()`? 😄